src/gui/gui.cc

```diff
@@ -1373,6 +1374,20 @@ query using a REST api. See the wiki for details.
 The debugger might be required in some cases.)"));
         changed |=
             ImGui::InputInt(_("Web Server Port"), &debugSettings.get<Emulator::DebugSettings::WebServerPort>().value);
+        if (ImGui::Checkbox(_("Enable SIO1 Server"), &debugSettings.get<Emulator::DebugSettings::SIO1Server>().value)) {
+            changed = true;
+            if (debugSettings.get<Emulator::DebugSettings::SIO1Server>()) {
+                g_emulator->m_sio1Server->startServer(&g_emulator->m_loop,
+                                                     debugSettings.get<Emulator::DebugSettings::SIO1ServerPort>());
+            } else {
+                g_emulator->m_sio1Server->stopServer();
+            }
+        }
+        ShowHelpMarker(_(R"(This will activate a tcp server, that will
+relay information between tcp and sio1.
+See the wiki for details.)"));
```

Comment:
![image](https://user-images.githubusercontent.com/7281574/152095746-66a40608-e9e4-4e95-ab45-99031cc8412d.png)

Reply:
That last tcp should say redux, derp.

Reply:
Also, note to self, add wiki entry.